src/Symfony/Component/VarDumper/Tests/Caster/FiberCasterTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Tests\Caster;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\VarDumper\Test\VarDumperTestTrait;
+
+/**
+ * @requires PHP 8.1
+ */
+class FiberCasterTest extends TestCase
+{
+    use VarDumperTestTrait;
+
+    public function testCastFiberNotStarted()
+    {
+        $fiber = new \Fiber(static function () {
```

Comment:
`$fiber = new \Fiber(static fn() => true);`

Reply:
I actually wanted to, but even with `@requires PHP 8.1`, I had errors on PHP 7.2 CI that was broken when using short functions :confused:

Reply:
ah indeed!